backend/rumergy_backend/rumergy/serializers/data_point_serializer.py

```diff
@@ -0,0 +1,11 @@
+from rest_framework import serializers
+from rumergy_backend.rumergy.models.data_point import DataPoint
+# from rumergy_backend.rumergy.models.model import Model
+
+class DataPointSerializer(serializers.ModelSerializer):
+    
+    # model = serializers.PrimaryKeyRelatedField(queryset=Model.objects.all())
```

Comment:
Should be `many=False` I think

Reply:
Model relationship doesn't go here, goes in model's serializer